src/lines-yardstick.coffee

```diff
@@ -19,8 +20,8 @@ class LinesYardstick
     targetTop = pixelPosition.top
     targetLeft = pixelPosition.left
     defaultCharWidth = @model.getDefaultCharWidth()
+    return Point(0, 0) if targetTop < 0
```

Comment:
Interesting. Why are we returning early here? Seems like the routine below can handle it, even if the cost is a little higher. On the other hand, performance will be consistent in all scenarios.